### PR TITLE
removed from mysql package

### PR DESCRIPTION
**Pull request by alphmth**

### CODE REVIEW

File: packages/mysql/src/index.js [hunk 1]
```diff
@@ -19,7 +21,62 @@ class KeyvMysql extends KeyvSql {
 			.then(connection => sql => connection.execute(sql)
 				.then(data => data[0]));
 
-		super(options);
+		this.opts = Object.assign({
+			table: 'keyv',
+			keySize: 255,
+		}, options);
+
+		const createTable = `CREATE TABLE IF NOT EXISTS ${this.opts.table}(id VARCHAR(${Number(this.opts.keySize)}) PRIMARY KEY, value TEXT )`;
```
Comment (stropicall) [inline packages/mysql/src/index.js:29]: @jaredwray `key` is a keyword in mysql so I changed it to `id`

**Comment by jaredwray** [comment]
@alphmth - looks like it is failing the testing. Are you able to resolve the issue?

**Comment by jaredwray** [comment]
@alphmth - can you do a pull request for each replacement. One pull request for mysql and another pull request for postgres?

**Comment by alphmth** [comment]
@jaredwray `Error: Cannot find module '/home/runner/work/keyv/keyv/node_modules/@keyv/test-suite/dist/index.js'. Please verify that the package.json has a valid "main" entry`

I think it's not building `@keyv/test-suite` module in CI.

**Pull request merged**